### DISCUSSION TITLE
STBS For SideFX Houdini?

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [STORM-IRIT/Radium-Engine#841] [Q&A] [answered]
Hi, is it possible to "#include" the STBS-and-related source files in a Houdini SOP plug-in, or are they written to be used only in a Radium-based App?

**Comment by dlyr** [comment, marked as answer]
Hi,
As soon as you respect the [license](https://github.com/STORM-IRIT/Radium-Engine/blob/master/LICENSE), you are free to use it.
A reference to Radium-Engine will be appreciated ([see here](https://github.com/STORM-IRIT/Radium-Engine#readme) for how to site)
